SWLOR.Game.Server/Feature/AbilityDefinition/Devices/IncendiaryBombAbilityDefinition.cs

```diff
@@ -18,6 +18,7 @@ private static void ApplyEffect(uint creature, int dmg)
             var attackerStat = GetLocalInt(OBJECT_SELF, "DEVICE_ACC");
             var attack = GetLocalInt(OBJECT_SELF, "DEVICE_ATK");
             dmg += GetLocalInt(OBJECT_SELF, "DEVICE_DMG");
+            dmg += attackerStat / 2;
```

Comment:
Instead of doing this on every AOE, you should do it one time in ExplosiveAOEImpact inside ExplosiveBaseAbilityDefinition. This will let you change it in one place instead of every file.